projects/ngx-device-detector/src/lib/device-detector.service.ts

```diff
@@ -111,6 +111,8 @@ export class DeviceDetectorService {
       : this.isDesktop(this.userAgent)
       ? DeviceType.Desktop
       : DeviceType.Unknown;
+
+    this.os = this.device === iPad ? Constants.OS.IOS : this.os;
```

Comment:
Is this the best solution we can have?
Looks like a specific case condition. Wouldn't it be the same for iPhones as well?

Reply:
Yes, you are right, now I did it a better way, please check.

Reply:
LGTM! 👍 Good stuff!